DOCS: Rewrite "Install OpenVINO Development Tools" page - port to 22.1 (#13862)
openvinotoolkit/openvino: docs/install_guides/installing-model-dev-tools.md

# Install OpenVINO™ Development Tools {#openvino_docs_install_guides_install_dev_tools}

OpenVINO Development Tools is a set of utilities that make it easy to develop and optimize models and applications for OpenVINO. It provides the following tools:

* Model Optimizer
* Benchmark Tool
* Accuracy Checker and Annotation Converter
* Post-Training Optimization Tool
* Model Downloader and other Open Model Zoo tools

The instructions on this page show how to install OpenVINO Development Tools. If you are a Python developer, it only takes a few simple steps to install the tools with PyPI. If you are developing in C++, OpenVINO Runtime must be installed separately before installing OpenVINO Development Tools.

In both cases, Python 3.6 - 3.9 need be installed on your machine before starting.

> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI.

## <a name="python-developers"></a>For Python Developers

> If you are a Python developer, follow the steps in the <a href="openvino_docs_install_guides_install_dev_tools.html#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.

Reviewer: ```suggestion
If you are a Python developer, follow the steps in the <a href="o#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.
```

Author: > 

I do not follow this change, could you elaborate it, please?

Reviewer: openvino_docs_install_guides_install_dev_tools.html is the current file, right? we don't need to add the file name when referencing to an anchor in the same file

Author: Well, normally yes. We will see if it works during the tests ;)

  After you have installed OpenVINO Runtime from an archive file, you can find a set of requirements files in the <INSTALL_DIR>\tools\ directory. The requirements files will install the matching version of OpenVINO Development Tools and its dependencies.

  1. Install the OpenVINO Development Tools mandatory requirements using the following command:

   .. code-block:: sh

      pip install -r <INSTALL_DIR>\tools\requirements.txt

  2. If you are using additional frameworks, you must also install the requirements for those frameworks using the corresponding requirements file. For example, if you are using a TensorFlow model, use the following command to install requirements for TensorFlow:

   .. code-block:: sh

      pip install -r <INSTALL_DIR>\tools\requirements_tensorflow2.txt

  **Alternative: Install the openvino-dev Package from PyPI**
  You can also install OpenVINO Development Tools from PyPI using the following command.

   .. important::
    Make sure to specify the `openvino-dev` version that matches your installed version of OpenVINO Runtime. Otherwise, compatibility errors are likely to occur.

   .. code-block:: sh

>       pip install openvino-dev[EXTRAS]==2022.2

Reviewer: ```suggestion
      pip install openvino-dev[EXTRAS]==2022.1
```

  where the EXTRAS parameter specifies one or more deep learning frameworks via these values: caffe, kaldi, mxnet, onnx, pytorch, tensorflow, tensorflow2. Make sure that you install the corresponding frameworks for your models. For example:

   .. code-block:: sh

>       pip install openvino-dev[tensorflow2,onnx]==2022.2

Reviewer: ```suggestion
      pip install openvino-dev[tensorflow2,onnx]==2022.1
```

@endsphinxdirective

For more details on the openvino-dev PyPI package, see https://pypi.org/project/openvino-dev/.

### Step 4. Test the Installation

To verify the package is properly installed, run the command below (this may take a few seconds):

```sh
mo -h
```

You will see the help message for Model Optimizer if installation finished successfully. If you get an error, refer to the [Troubleshooting Guide](./troubleshooting.md) for possible solutions.

Congratulations! You finished installing OpenVINO Development Tools with C++ capability. Now you can start exploring OpenVINO's functionality through example C++ applications. See the "What's Next?" section to learn more!

## <a name="get-started"></a>What's Next?
Now that you've installed OpenVINO Runtime, you're ready to run your own machine learning applications! Learn more about how to integrate a model in OpenVINO applications by trying out the following tutorials.

@sphinxdirective
.. tab:: Get started with Python

>    Try the `Python Quick Start Example <https://docs.openvino.ai/2022.2/notebooks/201-vision-monodepth-with-output.html>`_ to estimate depth in a scene using an OpenVINO monodepth model in a Jupyter Notebook inside your web browser.

Reviewer: ```suggestion
   Try the `Python Quick Start Example <https://docs.openvino.ai/2022.1/notebooks/201-vision-monodepth-with-output.html>`_ to estimate depth in a scene using an OpenVINO monodepth model in a Jupyter Notebook inside your web browser.
```

   
   .. image:: https://user-images.githubusercontent.com/15709723/127752390-f6aa371f-31b5-4846-84b9-18dd4f662406.gif
      :width: 400

   Visit the :ref:`Tutorials <notebook tutorials>` page for more Jupyter Notebooks to get you started with OpenVINO, such as:
   

>    * `OpenVINO Python API Tutorial <https://docs.openvino.ai/2022.2/notebooks/002-openvino-api-with-output.html>`_

Reviewer: ```suggestion
   * `OpenVINO Python API Tutorial <https://docs.openvino.ai/2022.1/notebooks/002-openvino-api-with-output.html>`_
```

>    * `Basic image classification program with Hello Image Classification <https://docs.openvino.ai/2022.2/notebooks/001-hello-world-with-output.html>`_

Reviewer: ```suggestion
   * `Basic image classification program with Hello Image Classification <https://docs.openvino.ai/2022.1/notebooks/001-hello-world-with-output.html>`_
```

>    * `Convert a PyTorch model and use it for image background removal <https://docs.openvino.ai/2022.2/notebooks/205-vision-background-removal-with-output.html>`_

Reviewer: ```suggestion
   * `Convert a PyTorch model and use it for image background removal <https://docs.openvino.ai/2022.1/notebooks/205-vision-background-removal-with-output.html>`_
```
